torch/fx/experimental/symbolic_shapes.py

```diff
@@ -4096,22 +4098,15 @@ def issue_guard(guard: ShapeGuard) -> None:
 
             assert sources
             bounds = []
-            if r.lower != -sympy.oo:
+            if r.lower not in (-sympy.oo, -int_oo):
```

Comment:
in all these we could probably ask `r.lower.is_finite` but fine.

Reply:
Cannot, because `int_oo` reports `is_finite is True`, because you can't have an integer without it also being finite 😭

Reply:
This is why we can't have nice things